_posts/2017-05-07-Blockchain-Introduction.md

```diff
@@ -0,0 +1,422 @@
+  <div class="col-md-offset-3 col-md-6" style="padding: 0;">
+	{% include image.html img="/img/blockchain/partners.png" alt="Companies and consortiums" title="Companies and consortiums" caption="Table: IBM is part of Hyperledger and Microsoft is part of Enterprise Ethereum Alliance." %}
+</div>
+</div>
+
+
+# Thinking decentralized
+
+Last year, Ken Coenen gave a presentation about the popularity of APIs, and how companies team up to create innovative solutions.
+Data is freed from their silo's and made available through APIs.
+It's consumable for other departments and even other companies.
+However, when you think of it, all of this data is centralized and we need extra effort to expose it to other parties.
+
+When working with blockchain technology, your data is __decentralized by nature__.
+It's funny when you think of it...
+Why do we want to store the data somewhere centralized in a silo and then make an extra effort to expose it?
+Isn't is easier to start decentralized from the beginning and give access to the people who need it?
+What have we been doing all these years?
+
+We'll give an example.
+All applications implement their own user profile functionality.
+All of this user data - __your profile information__ - is duplicated across many companies.
+It's already a big improvement that applications allow you to use another platform's credentials.
+Logging in with your Facebook or Google account is becoming a habit.
+This gives the end user a way to minimize his/her digital footprint.
+Don Tapscott explains this really well in summer 2016's TED Talk [How the blockchain is changing money and business](https://youtu.be/og7PCYlDYsU?t=849).
+Of course, blockchain technology is still in its early stages.
+It's not even sure whether the technology will last.
+Although these statements are purely hypothetical, I find much food for thought in them.
+
+<div class="row" style="margin: 0 auto 2.5rem auto; width: 66%; padding: 0;">
+  <div class="col-md-offset-3 col-md-6" style="padding: 0;">
+	{% include image.html img="/img/blockchain/network-types.jpg" alt="Network types" title="Network types" %}
+</div>
+</div>
+
+
+# Conclusion
+
+When talking with people about the possibilities of blockchain, it quickly becomes clear that we still have a long way to go.
+People aren't waiting for yet another technological revolution.
+Instead, we need to start small.
+Blockchain and distributed ledger technology in general will have to evolve naturally.
+
+Blockchain solutions like IBM Blockchain or Microsoft Azure Blockchain-as-a-Service make the technology very accessible to companies in an early stage.
+We believe that a __private blockchain network__ is the __best way to start__ for a company because of the following reasons:
+
+* Throwing all your data at the world is still a very scary idea
+* You have to take all __legal aspects__ into account (think of the EU's new General Data Protection Regulation)
```

Comment:
Perhaps make the EU's regulation clickable and have it link to the actual regulation (if something like it exists)?
I assume there are plenty of people that are unaware of it.